innkeeper/client_test.go

```diff
@@ -296,6 +296,15 @@ func TestParsingInnkeeperComplexRoute(t *testing.T) {
 						"type": "STRICT"
 					}]
 				},
+                "predicates": [{
+                    "name": "customPredicate1"
+                }, {
+                    "name": "customPredicate2",
+                    "args": []
+                }, {
+                    "name": "customPredicate3",
+                    "args": ["string1", 3.14]
```

Comment:
do we really need float numbers? for filters we only integers (at least in Innkeeper)